web/web/models/user.py

```diff
@@ -77,29 +78,31 @@ def __repr__(self):
     # Relationships
     login = relationship('Login', backref=db.backref('user'), uselist=False)
 
+    notifications = relationship('Notification', backref=db.backref('user'))
+
+    alerts = relationship('Alert', backref=db.backref('user'))
+
 
-# Relationships on other tables
-Address.user = relationship('User',
-                            backref=db.backref('address'),
-                            uselist=False)
+##########################
+### MARSHMALLOW SCHEMA ###
+##########################
 
 
 class UserSchema(SQLAlchemyAutoSchema):
     roles = fields.Method('get_roles', dump_only=True)
-
     postal_code = fields.Method('get_postal_code', dump_only=True)
 
-    address = fields.Nested(AddressSchema(), load_only=True)
-
     # Marshmallow methods
     def get_roles(self, obj):
         roles = obj.get_roles()
         result_roles = []
         for role in roles:
             result_roles.append(role.name.value)
+            print(role.name.value)
         return result_roles
 
     def get_postal_code(self, obj):
+        print(obj.address)
```

Comment:
🔪

Reply:
Thanks will remove - let me know if there is anything else to fix!